cognite/powerops/custom_modules/power_model_v1/data_models/containers/Case.container.yaml

```diff
@@ -0,0 +1,28 @@
+space: '{{powerops_models}}'
+externalId: Case
+name: Case
+usedFor: node
+properties:
```

Comment:
Should we add the output of the preprocessor here (case file, cut file, reservoir cut mapping file, [extra files]) here, or where should those live?

Reply:
They are linked to the Scenario - do you think it makes more sense to put them here?

Reply:
Yes, we can't link them to the scenario - the scenario is a static config that should not change from day to day (or for every time we run SHOP/preprocessor), but a new Case is created for every time we run SHOP/preprocessor).